kie-gwthelper-maven-plugin/src/main/java/org/kie/maven/gwthelper/plugin/AddSourceMojo.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.maven.gwthelper.plugin;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.util.StringUtils;
+
+/**
+ * Add more source directories to the POM.
+ */
+@Mojo(name = "add-source", defaultPhase = LifecyclePhase.GENERATE_SOURCES, threadSafe = true)
+public class AddSourceMojo  extends AbstractMojo {
+
+    private final static String SRC_MAIN_JAVA = "/src/main/java".replace("/", System.getProperty("file.separator"));
```

Comment:
You can reuse the property name from https://docs.oracle.com/javase/8/docs/api/java/io/File.html#separator

Reply:
_done_

Reply:
Applying it to line 41 as well is the small detail lying between the current state and perfection :slightly_smiling_face:.